### PR TITLE
New host, who dis?

### LINKED ISSUE #139: [11/7] IRL Season 4 Website Updates
[IRL Season 4 Website Updates](https://bugzilla.mozilla.org/show_bug.cgi?id=1503963).

- [x] Replace host photo and blurb
- [x] Copy update to homepage
- [x] Copy update to footer
- [x] Update social media blurbs and images
- [ ] Go live Nov 7th

### PR DESCRIPTION
**Pull request by stephaniehobson**
Fix [bug 1503963](https://bugzilla.mozilla.org/show_bug.cgi?id=1503963).

Closes #139

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Needs to have the social media snippet updated.

**Comment by stephaniehobson** [comment]
We have a target launch date of Wednesday (any time) for this.

**Comment by stephaniehobson** [comment]
Updated!

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <meta property="og:image:width" content="{{ $.Scratch.Get "ogImageWidth" }}">
     <meta property="og:image:height" content="{{ $.Scratch.Get "ogImageHeight" }}">
     <meta property="og:title" content="{{ $.Scratch.Get "ogTitle" | default "IRL Podcast" }}">
-    <meta property="og:description" content="{{ .Page.Params.ogDescription | default "Veronica Belmont explores real stories of life online – and real talk about how we can all keep the Internet healthy, weird and wonderful. IRL is an original podcast from Mozilla Corporation." }}">
+    <meta property="og:description" content="{{ .Page.Params.ogDescription | default "Manoush Zomorodi explores real stories of life online – and real talk about how we can all keep the Internet healthy, weird and wonderful. IRL is an original podcast from Mozilla Corporation." }}">
```
Comment (craigcook) [inline layouts/_default/baseof.html]: Do we want to take this opportunity to add "the" before "Mozilla Corporation" for better grammar?

**Pull request merged**